fdbserver/worker.actor.cpp

```diff
@@ -1984,6 +1984,9 @@ ACTOR Future<Void> workerServer(Reference<IClusterConnectionRecord> connRecord,
 						            localInfo.blobManager.present() ? localInfo.blobManager.get().id() : UID())
 						    .detail("EncryptKeyProxyID",
 						            localInfo.encryptKeyProxy.present() ? localInfo.encryptKeyProxy.get().id() : UID());
+						if (localInfo.client.encryptKeyProxy != localInfo.encryptKeyProxy) {
```

Comment:
Maybe in another PR, but is it possible to remove the duplicated `encryptKeyProxy` from `DBServerInfo`? For example `CommitProxyInterface` is only presented in `ClientDBInfo` but not `ServerDBInfo`.

Reply:
Yeah i want to do this eventually as well, probably makes sense to tackle this in a separate pr